cpp/include/cudf/io/detail/parquet.hpp

```diff
@@ -66,7 +69,7 @@ class reader {
   /**
    * @brief Destructor explicitly-declared to avoid inlined in header
    */
-  ~reader();
```

Comment:
Is the reader derived somewhere?

Reply:
It will be derived in the next chunked reader. This is to prepare for it.

Reply:
That's right. Thanks!